deploy/automl-java-deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    io.kompose.service: automl-web
+  name: automl-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: automl-web
+  strategy: { }
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: automl-web
+    spec:
+      containers:
+        - image: kayc/automl:demo # change yourself image
```

Comment:
Please prepend the build step to the deploy step in the Readme Doc as well, then use the local image to deploy it to keep our deployment up to date.